mods/gen/weapons/missiles.yaml

```diff
@@ -602,13 +602,29 @@ missile.missile_defender.garrison.air:
 	Warhead@1Dam: SpreadDamage
 		Damage: 5000
 
-missile.missile_defender.locked.normal:
-	Inherits: missile.missile_defender.ground
-	InvalidTargets: Vehicle, Tank, Ship
+laser.missile_defender.locked.ground:
+	ReloadDelay: 4
+	Range: 4c384
+	ValidTargets: Vehicle, Tank, Ship, Structure
+	Projectile: LaserZap
+		Width: 30
+		ZOffset: 2047
+		Color: FF0000
+		Duration: 3
+
+laser.missile_defender.locked.air:
+	ReloadDelay: 4
+	Range: 6c0
+	ValidTargets: AirborneActor
+	Projectile: LaserZap
+		Width: 30
+		ZOffset: 2047
+		Color: FF0000
+		Duration: 3
 
 missile.missile_defender.locked.ground:
 	Inherits: missile.missile_defender.ground
-	ValidTargets: Vehicle, Tank, Ship
+	ValidTargets: Vehicle, Tank, Ship, Structure
```

Comment:
I checked the original game, doesn't seem like you can lock on buildings, just vehicles. Why add Structure here?

Reply:
really? then I will remove this